src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 96px;
```

Comment:
don't hardcode the `width`, what if you want only 4 or 6 stars 😉
batter use `margin`/`gap` to set space between starts

Reply:
I got the point! It even helped to pass all tests! Thanks :)